deepfence_frontend/apps/dashboard/src/features/secrets/pages/SecretScanResults.tsx

```diff
@@ -915,7 +932,9 @@ const BulkActions = ({
         content={
           <>
             <DropdownItem onClick={() => onTableAction(ids, ActionEnumType.UNMASK, '')}>
-              Un-mask secrets
+              {isHost
+                ? 'Un-mask vulnerabilities for this host'
```

Comment:
should be secret Un-mask vulnerabilities for this host --> Un-mask secrets for this host